### PR TITLE
Add docs for amazon ads destination

### PR DESCRIPTION
**Pull request by tcgilbert**
### Proposed changes

Add docs for amazon ads destination

### Merge timing
asap

### CODE REVIEW

File: src/connections/destinations/catalog/actions-amazon-amc/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+### Add the destination to your **Engage Space**. 
+
+
+1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank”} search for "Amazon Ads DSP and AMC".
+2. Select Amazon Ads DSP and AMC and click **Add Destination**.
+3. Select the **Engage Space** you'd like to add the destination to.
+4. Once added, view and input the settings you'd need to complete to configure the destination. 
+- **Connection**: Click "connect" to authenticate your destination with Amazon.
+- **Region**: Select the Amazon Region to deliver data to: NA, EU, or FE.
+- **Enable the destination**
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-amazon-amc/index.md:26]:
```suggestion
4. Once added, view and input the settings you'd need to complete to configure the destination. 
  - **Connection**: Click **Connect to [Destination Name]** to authenticate your destination with Amazon.
  - **Region**: Select the Amazon Region to deliver data to: NA, EU, or FE.
  - **Enable the destination**: Switch the toggle to on to enable your destination. 
```

File: src/connections/destinations/catalog/actions-amazon-amc/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+### Connect your Engage Audience(s) to the destination.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-amazon-amc/index.md:28]:
```suggestion
### Connect your Engage Audience(s) to the destination
```

File: src/connections/destinations/catalog/actions-amazon-amc/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+1. Navigate to the desired Audience in Engage, and select "Add Destination".
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-amazon-amc/index.md:30]:
```suggestion
1. Navigate to the desired Audience in Engage, and select **Add Destination**.
```

File: src/connections/destinations/catalog/actions-amazon-amc/index.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+2. Select the "Amazon Ads DSP and AMC" destination you configured just created.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-amazon-amc/index.md:31]:
```suggestion
2. Select the **Amazon Ads DSP and AMC** destination you just created.
```

File: src/connections/destinations/catalog/actions-amazon-amc/index.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+4. After adding to the Engage audience, click on the destination from the audience page to view and complete the [audience-specific settings](https://advertising.amazon.com/API/docs/en-us/amc-advertiser-audience#tag/Audience-Metadata){:target="_blank"}.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-amazon-amc/index.md:32]:
```suggestion
4. After adding your destination to the Engage audience, click on the destination from the audience page to view and complete the [audience-specific settings](https://advertising.amazon.com/API/docs/en-us/amc-advertiser-audience#tag/Audience-Metadata){:target="_blank"}. 
```

File: src/connections/destinations/catalog/actions-amazon-amc/index.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- **Advertiser ID**
+- [**Country Code**](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-cloud/audiences/audience-management-service#country-code){:target="_blank"}: A 2-character string in the ISO 3166 format that will be applied for all records within the audience. https://www.iso.org/obp/ui/#search 
+- (Optional) **CPM Cents**: Cost per thousand impressions (CPM) in cents. For example, $1.00 = 100 cents.
+- (Optional) **Currency**
+- **Description**: The audience description. Must be an alphanumeric, non-null string between 0 to 1000 characters in length.
+- **External Audience ID**: The user-defined audience identifier. This should be a unique, user-defined audience identifier (e.g., "audience-id-for-device").
+- **TTL**: Time-to-live in seconds. The amount of time the record is associated with the audience. 
+  - Values allowed: [0 .. 34300800 ] (e.g., 2592000 for 30 days, 34300800 for 397 days). 
+{% include components/actions-fields.html %}
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-amazon-amc/index.md:41]:
```suggestion

{% include components/actions-fields.html %}
```

File: src/connections/destinations/catalog/actions-amazon-amc/index.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+### Configure your Mappings
+
+1. Click on the destination from the audience page, and navigate to "Matching Mappings" from the destination side view.
+2. Click "Add mapping".
+3. Configure the mapping fields.
+4. Save and enable the mapping.
+5. Complete! When the audience is computed in Segment, you will see the audience created and records delivered to Amazon.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-amazon-amc/index.md:49]:
```suggestion
1. Click on the destination from the audience page, and navigate to **Matching Mappings** from the destination side view.
2. Click **Add mapping**.
3. Configure the mapping fields.
4. Save and enable the mapping.
5. Complete! When the audience is computed in Segment, you will see the audience created and records delivered to Amazon.
```

File: src/connections/destinations/catalog/actions-amazon-amc/index.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+{% include components/actions-fields.html %}
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-amazon-amc/index.md:51]: @tcgilbert It looks like this is also up above - did you want it on the bottom of the page, or above the Configure your mappings section?

File: src/connections/destinations/catalog/actions-amazon-amc/index.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+- [**Country Code**](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-cloud/audiences/audience-management-service#country-code){:target="_blank"}: A 2-character string in the ISO 3166 format that will be applied for all records within the audience. https://www.iso.org/obp/ui/#search 
+- (Optional) **CPM Cents**: Cost per thousand impressions (CPM) in cents. For example, $1.00 = 100 cents.
+- (Optional) **Currency**
+- **Description**: The audience description. Must be an alphanumeric, non-null string between 0 to 1000 characters in length.
+- **External Audience ID**: The user-defined audience identifier. This should be a unique, user-defined audience identifier (e.g., "audience-id-for-device").
+- **TTL**: Time-to-live in seconds. The amount of time the record is associated with the audience. 
+  - Values allowed: [0 .. 34300800 ] (e.g., 2592000 for 30 days, 34300800 for 397 days). 
+{% include components/actions-fields.html %}
+
+### Configure your Mappings
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-amazon-amc/index.md:43]:
```suggestion
### Configure your mappings
```

File: src/connections/destinations/catalog/actions-amazon-amc/index.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+4. After adding to the Engage audience, click on the destination from the audience page to view and complete the [audience-specific settings](https://advertising.amazon.com/API/docs/en-us/amc-advertiser-audience#tag/Audience-Metadata){:target="_blank"}. 
+- **Advertiser ID**
+- [**Country Code**](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-cloud/audiences/audience-management-service#country-code){:target="_blank"}: A 2-character string in the ISO 3166 format that will be applied for all records within the audience. https://www.iso.org/obp/ui/#search 
+- (Optional) **CPM Cents**: Cost per thousand impressions (CPM) in cents. For example, $1.00 = 100 cents.
+- (Optional) **Currency**
+- **Description**: The audience description. Must be an alphanumeric, non-null string between 0 to 1000 characters in length.
+- **External Audience ID**: The user-defined audience identifier. This should be a unique, user-defined audience identifier (e.g., "audience-id-for-device").
+- **TTL**: Time-to-live in seconds. The amount of time the record is associated with the audience. 
+  - Values allowed: [0 .. 34300800 ] (e.g., 2592000 for 30 days, 34300800 for 397 days).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-amazon-amc/index.md:40]: @tcgilbert do you have a blurb for advertiser ID/Currency?

```suggestion
  - **Advertiser ID**:
  - [**Country Code**](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-cloud/audiences/audience-management-service#country-code){:target="_blank"}: A 2-character string in the ISO 3166 format that will be applied for all records within the audience.
  - (Optional) **CPM Cents**: Cost per thousand impressions (CPM), in cents. For example, $1.00 = 100 cents.
  - (Optional) **Currency**:
  - **Description**: The audience description. Must be an alphanumeric, non-null string between 0 to 1000 characters in length.
  - **External Audience ID**: The user-defined audience identifier. This should be a unique, user-defined audience identifier (For example., "audience-id-for-device").
  - **TTL**: Time-to-live, in seconds. The amount of time the record is associated with the audience. Values allowed are 0 .. 34300800 (For example, 2592000 for 30 days, 34300800 for 397 days). 
```

File: src/connections/destinations/catalog/actions-amazon-amc/index.md (hunk 8 above)
Comment (tcgilbert) [reply src/connections/destinations/catalog/actions-amazon-amc/index.md:48]: removed the one above, and applied changes. thanks for review!

**Review by forstisabella — APPROVED** [review]

File: src/connections/destinations/catalog/actions-amazon-amc/index.md (hunk 10 above)
Comment (tcgilbert) [reply src/connections/destinations/catalog/actions-amazon-amc/index.md]: no blurb

File: src/connections/destinations/catalog/actions-amazon-amc/index.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+[Amazon Ads](https://advertising.amazon.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} can help you achieve your marketing goals, whether that's building brand awareness, driving sales, or increasing customer loyalty. 
+
+The Segment - Amazon Ads DSP and AMC integration allows users to connect their Engage Audiences to Amazon Ads to run ads based on certain attributes & audiences defined in Segment, like the people who have visited your site.
+ 
+This destination is maintained by Segment. For any issues with the destination, [contact the Segment Support team](mailto:friends@segment.com).
+
+## Getting started
+
+### Add the destination to your **Engage Space**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-amazon-amc/index.md:17]:
```suggestion
### Add the destination to your Engage Space. 
```

**Review by forstisabella — APPROVED** [review]

**Pull request merged**